### ISSUE TITLE
Form field does not have a permanent label

### ISSUE THREAD

**Issue by jrbenn** [strategicdata/pmhc-mds-user-documentation#25]
The "Search docs" input field does not have a permanent label and rely on the "Search docs" placeholder. This may particularly disadvantage users that find navigating the web cognitively challenging. Users may struggle to understand the purpose and operation of the interactive elements. The temporary labels may confuse or misguide users.

Issue # 224 in https://docs.google.com/spreadsheets/d/13iTb4aW7KulDcbj5Q2xD78H4KOrMmhvfIDW1JMffY7o/edit?gid=155984173#gid=155984173
